cypress.d.ts

```diff
@@ -217,12 +217,12 @@ declare global {
 			/**
 			 * Set the read status of all emails to read.
 			 */
-			mailpitSetAllEmailStatusAsRead(): Chainable<void>;
+			mailpitSetAllEmailStatusAsRead(): Chainable<string>;
 
 			/**
 			 * Set the read status of all emails to unread.
 			 */
-			mailpitSetAllEmailStatusAsUnRead(): Chainable<void>;
+			mailpitSetAllEmailStatusAsUnRead(): Chainable<string>;
```

Comment:
Changing the return signature is a breaking change which is something not needed at the moment.

Reply:
I changed the signature here because the implementation im MailpitCommands.ts in Version 1.3 returns Cypress.Chainable<Cypress.Response> so there was a mismatch. There was a mismatch between declaration and implementation. I didn't changed the implementation here. I can undo the im declaration change but the implementation schould then be changed to return the correct type?


`	mailpitSetAllEmailStatusAsUnRead(): Cypress.Chainable<string> {
		return this.setReadStatus(null, false);
	}`

Reply:
Is it safe to have this missmatch?